docs/cow-protocol/tutorials/solvers/onboard.md

```diff
@@ -0,0 +1,159 @@
+
+## 7. Weekly rewards
+Every week on Tuesday your solver will receive rewards for settling auctions. These are distributed in COW tokens to the rewards address you provided. Rewards in COW will be sent to that address on mainnet, and reimbursements for slippage will be distributed on the same network as where the solving took place.
+
+We advise using a single rewards address that is available on all networks.
+
+[You can find more information about how we calculate and distribute rewards here.](https://docs.cow.fi/cow-protocol/reference/core/auctions/rewards)
+
+## 8. Slippage accounting
+Solvers are able to use the settlement contract's buffers when settling auctions. This can often result in one of two situations when prices move during an auction:
+
+1. The settlement contract receives more tokens than what was promised to the user.
+
+In this situation, some extra tokens will be left in the settlement contract after the auction is finished.
+
+2. The settlement contract receives less tokens than what was promised to the user.
+
+In this situation, if the settlement contract has enough of the tokens in its buffers, then the user will still receive the amount of tokens that they were promised. But, some of the funds will have been borrowed from the settlement contract's buffers.
+
+These will be converted to native currencies (ETH or xDAI) and transferred to your solver at the end of the week. If the settlement contract received more tokens in total over the week than what was borrowed, then you will receive this amount to your rewards address. But if more was borrowed from the settlement contract than what was returned, then your solver will be asked to reimburse the difference.
+
+## 9. Moving to other networks
+After joining the solver competition on Arbitrum we can enable your solver on Gnosis chain and Base network relatively soon. For this we will go through the same process of generating addresses, vouching for them, and whitelisting them. For each new chain that we deploy your solver on we will need:
+
+- A rewards address for receiving the weekly COW rewards associated with that network
+- An endpoint that you would like to use for staging ad production
+
+## 10. Joining the mainnet competition
+We usually wait a bit before migrating solvers to mainnet. After your solver has been active on other networks for some time the team can enable your solver on mainnet. 
+
+## 11. Joining the quoting competition
+Besides the solving competition, there is also a quoting competition. This involves responding to a single order batch with a proposed execution for the order and a price that could be delivered to the user. 
+
+If your solver provides the best price for the user, and that quote then leads to an order that is settled on chain. Then your solver will receive a fixed reward for providing this quote, regardless of which solver won the associated auction and settled the order on chain.
```

Comment:
I would remove this sentence completely (as it is inaccurate) and just keep the sentence below that simply links to the section that describes the quote rewards